### PR TITLE
Remaining time estimation 

### PR DESCRIPTION
**Pull request by therealhoodboy**
Added new indicators how lang your resources will last  
Further improved RessourceMenu

### CODE REVIEW

File: src/components/RessourceMenu/RessourceMenu.tsx [hunk 1]
```diff
@@ -1,47 +1,74 @@
 import { useStore } from '../../store/store';
+
 import './RessourceMenu.scss';
 
+
+
+//const test = Math.floor(ressources.gold)
+
```
Comment (Resaki1) [inline src/components/RessourceMenu/RessourceMenu.tsx:8]: I think these can be removed

File: src/components/RessourceMenu/RessourceMenu.tsx [hunk 2]
```diff
@@ -1,47 +1,74 @@
...
 export const RessourceMenu = () => {
   const ressources = useStore((state) => ({ ...state.ressources }));
   const buildingOutputs = useStore((state) => state.buildingOutputs);
   const reset = useStore((state) => state.reset);
 
+
+  const valueSmoothener: (m: number) => number = function (
```
Comment (Resaki1) [inline src/components/RessourceMenu/RessourceMenu.tsx:15]: could you give the parameters meaningful names? makes the code easier to read 🙂 
same for the remainingRessource function

File: src/components/RessourceMenu/RessourceMenu.tsx [hunk 3]
```diff
@@ -1,47 +1,74 @@
...
+
+    return (Math.round((m * 6000) / 100))
+  }
+
+
+  const remainingRessources: (v: number, t: number) => any = function (
+    v: number,
+    t: number,
+
+  ): any {
```
Comment (Resaki1) [inline src/components/RessourceMenu/RessourceMenu.tsx:26]: I think the return type here could be string instead of any

**Review by Resaki1 — CHANGES_REQUESTED** [review]
thanks!

File: src/components/RessourceMenu/RessourceMenu.tsx (hunk 3 above)
Comment (therealhoodboy) [reply src/components/RessourceMenu/RessourceMenu.tsx:26]: Thank you for your feedback 🥇

**Review by Resaki1 — APPROVED** [review]

**Pull request merged**